Directory.Packages.props

```diff
@@ -12,11 +12,11 @@
     <PackageVersion Include="Google.Protobuf" Version="[3.19.4,4.0)" />
     <PackageVersion Include="Grpc" Version="[2.44.0,3.0)" />
     <PackageVersion Include="Grpc.Net.Client" Version="[2.52.0,3.0)" />
-    <PackageVersion Include="Grpc.Tools" Version="[2.44.0,3.0)" />
+    <PackageVersion Include="Grpc.Tools" Version="[2.56.2,3.0)" />
     <PackageVersion Include="Microsoft.AspNetCore.Http.Abstractions" Version="[2.1.1,6.0)" />
     <PackageVersion Include="Microsoft.AspNetCore.Http.Features" Version="[2.1.1,6.0)" />
     <PackageVersion Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="[3.3.3]" />
-    <PackageVersion Include="Microsoft.CodeCoverage" Version="[17.4.1]" />
+    <PackageVersion Include="Microsoft.CodeCoverage" Version="[17.6.3]" />
```

Comment:
I think you should change the powershell script used by code coverage. Internal path was changed

Reply:
That shouldn't be the case as the script does not look for a hard-coded path.